vagrant/provisioning/roles/saml-config/tasks/main.yml

```diff
@@ -0,0 +1,71 @@
+- name: spring-config-saml.properties file
+  become: yes
+  become_user: arkcase
+  template:
+    src: spring-config-saml.properties
+    dest: "{{ root_folder }}/data/arkcase-home/.arkcase/acm/spring/spring-config-saml.properties"
+    backup: yes
+
+- name: read current setenv.sh
+  become: yes
+  become_user: arkcase
+  command: cat {{ root_folder }}/app/arkcase/bin/setenv.sh
+  register: setenv_contents
+  changed_when: false
+
+- name: replace ldap profile with ssoSaml
+  become: yes
+  become_user: arkcase
+  replace:
+    backup: yes
+    path: "{{ root_folder }}/app/arkcase/bin/setenv.sh"
+    regexp: "ldap"
+    replace: "ssoSaml"
+  when: '"ldap" in setenv_contents.stdout'
+
+- name: ensure ssoSaml is in the profile (other profiles already exist)
+  become: yes
+  become_user: arkcase
+  replace:
+    backup: yes
+    path: "{{ root_folder }}/app/arkcase/bin/setenv.sh"
+    regexp: "spring.profiles.active="
+    replace: "spring.profiles.active=ssoSaml"
+  when: '"ssoSaml" not in setenv_contents.stdout and "ldap" not in setenv_contents.stdout and "spring.profiles.active" in setenv_contents.stdout'
+
+- name: ensure ssoSaml is in the profile (no other profiles already exist)
+  become: yes
+  become_user: arkcase
+  lineinfile:
+    backup: yes
+    path: "{{ root_folder }}/app/arkcase/bin/setenv.sh"
+    line: 'JAVA_OPTS="$JAVA_OPTS -Dspring.profiles.active=ssoSaml"'
+  when: '"spring.profiles.active" not in setenv_contents.stdout'
+
+- name: update spring-security-config-saml.xml
+  become: yes
+  become_user: arkcase
+  replace:
+    backup: yes
+    path: "{{ root_folder }}/data/arkcase-home/.arkcase/acm/spring-security/spring-security-config-saml.xml"
+    regexp: "{{ item.regexp }}"
+    replace: "{{ item.replace }}"
+  loop:
+    - regexp: 'arkcase/acm/private/saml-keystore'
+      replace: 'arkcase/acm/private/keystore'
+    - regexp: 'type="java.lang.String" value="password"'
+      replace: 'type="java.lang.String" value="changeit"'
+    - regexp: 'key="armedia" value="AcMd3v\$"'
+      replace: 'key="eeoc-pre-prod-app-server" value="changeit"'
+    - regexp: 'type="java.lang.String" value="armedia"'
+      replace: 'type="java.lang.String" value="eeoc-pre-prod-app-server"'
+
+- name: copy the saml keystore
+  become: yes
+  become_user: arkcase
+  copy:
+    backup: yes
+    src: "{{ root_folder }}/install/arkcase/keystore"
```

Comment:
not sure how this keystore gets created yet; this is the pre-existing one that EEOC pre-prod already had.  Will figure out how to create this on EEOC production.